index.html

```diff
@@ -237,21 +234,31 @@ <h1>About <br />Myself</h1>
             <div>CSS</div>
           </li>
         </ul>
-        <div class="dropdown"><li class="section_header">Frameworks</li><span>^</span></div>
-        <div class="dropdown"> <li class="section_header">Skills</li><span>^</span>
-      </ul>
+        <div class="dropdown">
+          <li class="section_header">Frameworks</li><span>^</span>
+        </div>
+        <div class="dropdown">
+          <li class="section_header">Skills</li><span>^</span>
+      </ul></div>
+      
     </section>
+    <section class="footer_section">
     <footer class="footer">
-        <form action="https://formspree.io/f/mjvzpbgj" method="post">
-            <h1 class="form_header">Contact me</h1>
-            <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-            <div class="form_inputs">
-                <input type="text" name="name" placeholder="Name" class="form_input" maxlength="30" required class="form_input">
-                <input type="email" name="email" placeholder="Email" class="form_input" required class="form_input">
-                <textarea name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here" class="form_input" required></textarea>
-            </div>
-          <button type="submit" class="form_button">Get in Touch</button>
+      <form action="https://formspree.io/f/mjvzpbgj" method="post">
+        <h1 class="form_header">Contact me</h1>
+        <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you
+          need built or a project that needs coding. I’d love to help with it </p>
+        <div class="form_inputs">
+          <input type="text" name="name" placeholder="Name" class="form_input" maxlength="30" required
+            class="form_input">
+          <input type="email" name="email" placeholder="Email" class="form_input" required class="form_input">
+          <textarea name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here"
+            class="form_input" required></textarea>
+             <button type="submit" class="form_button">Get in Touch</button>
+        </div></section>
+       
```

Comment:
- [ ] [Optional] As per the project requirement you should stick to the design as much as possible that is: `color, font, image, text, and margin. please see the image below for reference. please see the border.
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/199729896-4b8da896-6eb8-4a17-93b6-c65533ce5f9d.png)
### Your design
![yourrrrrrrrr](https://user-images.githubusercontent.com/81727988/199729958-04848a35-7df6-4e8e-9d28-cf113adff2f2.png)